src/main/java/org/fundacionjala/at15/katas/langtonant/diana/Main.java

```diff
@@ -0,0 +1,27 @@
+package org.fundacionjala.at15.katas.langtonant.diana;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        System.out.println("WELCOME TO THE LANTONAN GAME!");
+        System.out.println("Crear tamaño del tablero");
```

Comment:
Sorry, I think I missed these updates in my previous review. Please change messages printed to english

Reply:
no Jose, it was something i added to the last one, sorry I missed it.